L1Trigger/TrackTrigger/interface/L1TrackQuality.h

```diff
@@ -43,6 +46,7 @@ class L1TrackQuality {
   // Passed by reference a track without MVA filled, method fills the track's MVA field
   void setL1TrackQuality(TTTrack<Ref_Phase2TrackerDigi_>& aTrack);
 
+  float runEmulatedTQ(std::vector<ap_fixed<10,5>> inputFeatures);
```

Comment:
Add a comment above to describe what this does. What is the purpose of this function if it is not used? I see in L1TrackQuality.cc that you define the function but then write out the insides of the function when actually doing inference and never using the function. Is there a reason for this?

Reply:
This function is used here: https://github.com/cms-L1TK/cmssw/blob/87c9532ce38218f2c445c03f288ef68d7ae44525/L1Trigger/TrackFindingTracklet/plugins/ProducerKFout.cc#L295 to run the TQ within the KFout step, as the code is set up the way it is, I can't pass a TTTrackWord to the setL1TrackQuality as we have before as I don't have a TTTrackWord to pass and I need the output so I can then set some bits, this is an ugly solution to this problem but I don't have a way around it